docroot/sites/settings/cgov_caching.settings.php

```diff
@@ -23,22 +23,31 @@
   $config['system.performance']['js']['preprocess'] = TRUE;
   $config['system.performance']['js']['gzip'] = TRUE;
 
-  switch ($env) {
-    case 'dev':
-    case '01dev':
-    case 'int':
-    case '01test':
-    case 'test':
-    case '01live':
-    case 'ode':
-      // Cache settings.
-      $config['system.performance']['cache']['page']['max_age'] = 16588800;
-      break;
+  // MEO environments.
+  if ($_ENV['AH_ENVIRONMENT_TYPE'] === 'meo') {
+    switch ($env) {
```

Comment:
This is messy. We should re-work this so it is just 1 switch. Maybe just fake the meo environment names...

But also, what in here is NOT going to get the max_age? Maybe we negate this whole thing. It looks like this is just for the everything but the local env.

(NOW, it USED to have different settings, but I think changes there make this no longer valid.)